vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogbundleinitialization.rb

```diff
@@ -16,12 +16,28 @@
 #
 # Inputs: $evm.root['service_template_provision_task'].dialog_options
 #
+# array_entry example - Classification::1
+def vmdb_object_from_array_entry(entry)
+  model, id = entry.split("::")
+  $evm.vmdb(model, id.to_i) if model && id
+end
+
+def array_value(hash, hash_key, value)
+  if hash.key?(hash_key)
+    values = hash[hash_key].split(',')
+    values << value
+    values.map { |str| "#{str}" }.join(',')
+  else
+    value
+  end
+end
 
 # Description: Look for service dialog variables in the root object that start with "dialog_option_[0-9]",
 def get_options_hash(dialog_options)
   # Setup regular expression for service dialog tags
-  options_regex = /^(dialog_option|dialog_tag)_(\d*)_(.*)/i
-  options_hash = {}
+  options_regex       = /^(dialog_option|dialog_tag)_(\d*)_(.*)/i
```

Comment:
If there is no number, then this would match on `"dialog_option__stuff"`.  Is that right?